sql/plan/insert.go

```diff
@@ -63,7 +63,7 @@ type InsertInto struct {
 	Source                sql.Node
 	ColumnNames           []string
 	IsReplace             bool
-	HasUnspecifiedAutoInc bool
+	UnspecifiedAutoIncIdx int
```

Comment:
Similar naming comment here... Using the same, self-documenting name consistently (including methods like `WithUnspecifiedAutoInc`) will go a long ways to making this code easier for other people to understand. `firstGeneratedAutoIncRowIdx` is my vote. 😄